lib/reporters/mini.js

```diff
@@ -28,6 +28,13 @@ function MiniReporter(options) {
 
 	this.options = Object.assign({}, options);
 
+	if (this.options.disableColors) {
+		chalk.enabled = false;
+		Object.keys(colors).forEach(function (key) {
```

Comment:
I just re-used the existing code that in other places did this but with "=true". But can make changes if you think that`s better.

https://github.com/ThomasBem/ava/blob/7c00c13844da6ab116676027b6482a57cf0f102a/lib/reporters/verbose.js#L9:L11

https://github.com/ThomasBem/ava/blob/7c00c13844da6ab116676027b6482a57cf0f102a/lib/reporters/mini.js#L14:L16